modules/sdk/project-templates/project-templates/src/main/java/com/liferay/project/templates/internal/util/JakartaCompatabilityUtil.java

```diff
@@ -5,17 +5,24 @@
 
 package com.liferay.project.templates.internal.util;
 
+import com.liferay.petra.string.StringPool;
```

Comment:
Sooo, `petra` is not really a great citizen when it comes to being used as a library outside of the actual DXP runtime (e.g. it doesn't always declare its true runtime dependencies, and can require special jvm args to enable reflection). If we're only using it for string constants, let's just make our own instead of adding a dependency on `petra`.

As an example, I made an effort a while back to expunge `petra` from Liferay Workspace: https://github.com/liferay-devtools/liferay-portal/commit/2cff1cd90c70a. Otherwise, it's kind of a minefield once you start running JDK21.